.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature request
+about: Suggest an idea for this project or its docs
+title: ''
+labels: kind/feature
```

Comment:
You don't have labels attribute in the document above. Why do you have it here? Alsi why `kind/`?
Is the `/kind feature` in the bottom not enough?

Reply:
Good question. This section cannot be modified by the user, i.e., you clicked "Feature Request", the issue will contain this label. The `/kind feature` can be removed but also shows how to interact with the bot.